<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据管理系统</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>631</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QPushButton" name="manageBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>商品管理</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="chartBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>销售统计</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="passwordBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>修改密码</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>70</y>
     <width>700</width>
     <height>410</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="managePage">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>700</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>商品管理</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QToolBox" name="toolBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>321</width>
       <height>360</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="toolBoxPage1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>317</width>
        <height>278</height>
       </rect>
      </property>
      <attribute name="label">
       <string>出售商品</string>
      </attribute>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>金额：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="sellCancelBtn">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>250</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>50</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>类型：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>87</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>品牌：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="sellTypeComboBox">
       <property name="geometry">
        <rect>
         <x>63</x>
         <y>50</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QComboBox" name="sellBrandComboBox">
       <property name="geometry">
        <rect>
         <x>63</x>
         <y>87</y>
         <width>210</width>
         <height>20</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QPushButton" name="sellOkBtn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>250</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>160</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>数量：</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="sellNumSpinBox">
       <property name="geometry">
        <rect>
         <x>63</x>
         <y>160</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="sellSumLineEdit">
       <property name="geometry">
        <rect>
         <x>63</x>
         <y>200</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>200</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>元</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>120</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>单价：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="sellPriceLineEdit">
       <property name="geometry">
        <rect>
         <x>63</x>
         <y>120</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>120</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>元</string>
       </property>
      </widget>
      <widget class="QLabel" name="sellLastNumLabel">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>160</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>剩余数量：000</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>已有商品入库</string>
      </attribute>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>50</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>类型：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>87</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>品牌：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>120</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>单价：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>160</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>数量：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="goodsOkBtn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>250</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
      <widget class="QPushButton" name="goodsCancelBtn">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>250</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
      <widget class="QComboBox" name="goodsTypeComboBox">
       <property name="geometry">
        <rect>
         <x>63</x>
         <y>50</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="goodsPriceLineEdit">
       <property name="geometry">
        <rect>
         <x>63</x>
         <y>120</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="goodsBrandComboBox">
       <property name="geometry">
        <rect>
         <x>63</x>
         <y>87</y>
         <width>210</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>120</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>元</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="goodsSumLineEdit">
       <property name="geometry">
        <rect>
         <x>63</x>
         <y>200</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>179</x>
         <y>200</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>元</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>金额：</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="goodsNumSpinBox">
       <property name="geometry">
        <rect>
         <x>63</x>
         <y>160</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>新商品入库</string>
      </attribute>
      <widget class="QSpinBox" name="newNumSpinBox">
       <property name="geometry">
        <rect>
         <x>63</x>
         <y>160</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>120</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>单价：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>160</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>数量：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="newPriceLineEdit">
       <property name="geometry">
        <rect>
         <x>63</x>
         <y>120</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="newTypeComboBox">
       <property name="geometry">
        <rect>
         <x>63</x>
         <y>50</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>50</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>类型：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>120</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>元</string>
       </property>
      </widget>
      <widget class="QPushButton" name="newCancelBtn">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>250</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>87</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>品牌：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="newOkBtn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>250</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="newBrandlineEdit">
       <property name="geometry">
        <rect>
         <x>63</x>
         <y>87</y>
         <width>210</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="newSumLineEdit">
       <property name="geometry">
        <rect>
         <x>62</x>
         <y>200</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>179</x>
         <y>200</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>元</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>金额：</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QListWidget" name="dailyList">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>321</width>
       <height>360</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="chartPage">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>700</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>销售统计</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="typeComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="updateBtn">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>更新显示</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
